doc/modules/partial_dependence.rst

```diff
@@ -1,21 +1,25 @@
 
 .. _partial_dependence:
 
-========================
-Partial dependence plots
-========================
+==================
+Partial dependence
```

Comment:
I think that we need  to change the name of the section

Maybe "Interaction between feature and target"

and then add 2 section PDP and ICE